pkg/addons/data/Deployment.apps/openshift-azure-branding/branding.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: branding
+  name: branding
+  namespace: openshift-azure-branding
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: branding
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: branding
+    spec:
+      containers:
```

Comment:
@y-cote need to copy the podAntiAffinity spec from the other console pods - please open an issue and action it

Reply:
https://github.com/openshift/openshift-azure/issues/1275